Formula/presto.rb

```diff
@@ -12,7 +12,7 @@ class Presto < Formula
   resource "presto-cli" do
     version cli_version
     url "https://search.maven.org/remotecontent?filepath=com/facebook/presto/presto-cli/#{version}/presto-cli-#{version}-executable.jar"
```

Comment:
OK, sure.
But I have a question. Where is Homebrew style?
I couldn't find Homebrew's coding terms.